### DISCUSSION TITLE
Cannot create playlist

### DISCUSSION THREAD

**Discussion by codeput** [rern/rAudio#773] [Fixed Issues]
I'm not able to add my personal playlist,.

**Comment by rern** [comment]
How did you add it?
If it's not a playlist created by **rAudio**, Addons > Import Playlists.

**Reply by codeput** [reply]
Thank You

**Reply by Vseason** [reply]
Is Addons > Import Playlists now broken or missing? I’m a new user, but on 20220610 I can’t get any rAudio-created playlist to reload, nor does there currently seem to be any way to import M3U files.

EDIT: After more poking around this morning, I finally realized it was indeed possible to reload an rAudio-created playlist by tapping on the playlist ICON, rather than the name. I’ve still not figured out any way to load or use existing M3U playlists though, so if anyone knows the answer I would appreciate a pointer or two. Thank you!

**Comment by ceejayemm** [comment]
After updating to 20220610 (or may be an earlier update) I found that /srv/http/playlists was owned by http:http rather than mpd:audio.  Changing the ownership allowed me to store my playlists. Please note I use the 'Shareddata' option so this might be a result of using this.

Chris
